pyOCD/tools/pyocd.py

```diff
@@ -648,7 +645,7 @@ def handle_clock(self, args):
         else:
             nice_freq = "%d Hz" % freq_Hz
 
-        print "Changed %s frequency to %s" % (swd_jtag, nice_freq)
+        print "Changed frequency to %s" % nice_freq
```

Comment:
I agree, switching to python 3 prints should be done.  Do you mind if this is done in future PR?  I would like to have one big PR which goes through and makes everything python 3 compatible.